src/app/components/shared/Button.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+interface childrenTypes {
+  children: React.ReactNode;
+  outlined?: boolean;
+  size?: "sm" | "lg" | "xs";
+}
+
+function Button({ children, outlined, size }: childrenTypes) {

```

Comment:
I feel this component's flexibility is a bit constrained; the size options are limited (e.g. "sm" just reduces the height but not the width). I would recommend to either add more flexibility or re-scope it to be a private component in the AuthModal.tsx file